native/Cargo.lock

```diff
@@ -926,7 +926,7 @@ dependencies = [
 
 [[package]]
 name = "datafusion-comet"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
This should have been updated as part of https://github.com/apache/datafusion-comet/pull/1404 but I missed this